src/hugr.rs

```diff
@@ -121,10 +121,10 @@ impl Hugr {
                         PortStyle::new(html_escape::encode_text(&format!("{}", ty)))
                     }
                     EdgeKind::StateOrder => match self.graph.port_links(port).count() > 0 {
-                        true => PortStyle::Text("".to_string()),
+                        true => PortStyle::text("", false),
```

Comment:
Wont we still have graphviz boxes for these ports which just don't have any text in them?

Reply:
Yes, but they don't weight in the cell widths, and they do not have a border.

![image](https://github.com/CQCL-DEV/hugr/assets/121866228/a5fa6e92-03ce-47a8-92bb-2d2d17baf1ac)